windows.versions

```diff
@@ -1,15 +1,16 @@
 {
   "windows": [
     {
-      "latest": true,
       "version": "2.28.4",
       "build": "1",
       "fluent-bit": "1.9.9",
       "kinesis-plugin": "v1.10.1",
       "firehose-plugin": "v1.7.1",
-      "cloudwatch-plugin": "v1.9.1"
+      "cloudwatch-plugin": "v1.9.1",
       "openssl": "3.0.7",
-      "flexBison": "2.5.22"
+      "flexBison": "2.5.22",
+      "latest": true,
+      "stable": true
```

Comment:
I think the first release should have stable: false

Reply:
This is required by `sync` task which needs the stable version tag.

We can maybe add in change log that this is marked as stable but not been properly vetted yet.